db/migrate/20150107233846_create_oauth_access_grants.rb

```diff
@@ -0,0 +1,15 @@
+class CreateOauthAccessGrants < ActiveRecord::Migration
```

Comment:
They were already in master. The unshema gem goes and makes the migrations easier by adding a more verbose set of schema migrations. These files were generated by the tool. 
https://github.com/neopoly/unschema